### PR TITLE
DTSCCI-1334 - Add hint text paid amount

### PR DESCRIPTION
**Pull request by ed14537**
### JIRA link ###

https://tools.hmcts.net/jira/browse/DTSCCI-1334

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by rishikrsharma — APPROVED** [review]

**Review by velimd — APPROVED** [review]

**Review by AbuSalamHMCTS — APPROVED** [review]

File: src/main/modules/i18n/locales/en.json [hunk 1]
```diff
@@ -1077,7 +1077,8 @@
     "CCJ_PAID_AMOUNT": {
       "AMOUNT_PAID": "Amount already paid",
       "PAGE_TITLE": "Has the defendant paid some of the amount owed?",
-      "TITLE": "Has the defendant paid some of the amount owed?"
+      "TITLE": "Has the defendant paid some of the amount owed?",
+      "AMOUNT_PAID_HINT": "The system only allows interest to be calculated on the full claim amount. If you have received any part payment, please use form N225 (insert link), where you can calculate the correct interest, and send to*  contactocmc@justice.gov.uk."
```
Comment (nigeldunne) [inline src/main/modules/i18n/locales/en.json:1081]: are you supposed to insert a link?

**Review by nigeldunne — COMMENTED** [review]
Add the text to the Wels file too, in English if you don't have the translation yet

File: src/main/modules/i18n/locales/en.json (hunk 1 above)
Comment (ed14537) [reply src/main/modules/i18n/locales/en.json]: no, just confirmed

**Review by gmikerlemos — DISMISSED** [review]

**Review by AbuSalamHMCTS — DISMISSED** [review]

**Review by peterpilgrim — COMMENTED** [review]
LGTM

File: src/main/modules/i18n/locales/en.json [hunk 2]
```diff
@@ -1077,7 +1077,8 @@
     "CCJ_PAID_AMOUNT": {
       "AMOUNT_PAID": "Amount already paid",
       "PAGE_TITLE": "Has the defendant paid some of the amount owed?",
-      "TITLE": "Has the defendant paid some of the amount owed?"
+      "TITLE": "Has the defendant paid some of the amount owed?",
+      "AMOUNT_PAID_HINT": "The system only allows interest to be calculated on the full claim amount. If you have received any part payment, please use form N225 https://www.gov.uk/government/publications/form-n225-request-for-judgment-and-reply-to-admission-specified-amount or https://assets.publishing.service.gov.uk/media/63f49b03e90e077bb3d7af45/n225a-eng-04-13_save.pdf, where you can calculate the correct interest, and send to contactocmc@justice.gov.uk."
```
Comment (nigeldunne) [inline src/main/modules/i18n/locales/en.json:1081]: https://design-system.service.gov.uk/components/text-input/

Avoid links
Do not include links within hint text. While screen readers will read out the link text when describing the field, they will not tell users that the text is a link.

**Review by anirudha-deshpande — DISMISSED** [review]

**Review by rishikrsharma — APPROVED** [review]

**Review by anirudha-deshpande — APPROVED** [review]

**Review by nigeldunne — APPROVED** [review]

**Pull request merged**